clients/src/main/resources/common/message/LeaderAndIsrRequest.json

```diff
@@ -29,11 +29,15 @@
   // Version 5 adds Topic ID and Type to the TopicStates, as described in KIP-516.
   //
   // Version 6 adds LeaderRecoveryState as described in KIP-704.
-  "validVersions": "0-6",
+  //
+  // Version 7 adds KRaft Controller ID field as part of KIP-866
+  "validVersions": "0-7",
   "flexibleVersions": "4+",
   "fields": [
     { "name": "ControllerId", "type": "int32", "versions": "0+", "entityType": "brokerId",
       "about": "The current controller ID." },
+    { "name": "KRaftControllerId", "type": "int32", "versions": "7+", "entityType": "brokerId",
```

Comment:
Can we add a reference to KIP-866 here?

Also, maybe good to set a default of -1 which will be used in the ZK case